app/views/users/sessions/verify_2fa.html.erb

```diff
@@ -1,4 +1,11 @@
 <h1>Two-factor authentication</h1>
+
+<script>
+  window.onload = function() {
```

Comment:
`on*`-style listeners should not be used in modern code (they have a lot of issues, for example, there can only be one at a time). Let's change it to the idiomaitc, fully supported, and newer `addEventListener('load', () => { ... })`. Same argument for switching from `function` to arrow function `() => {}` syntax.

Reply:
Ok.  There's a tiny bit of visible lag that I didn't see before, but if this is the correct way to do it now, I assume that's ok.  And maybe I only noticed it because I was watching for the focus.  Thanks for the education.